src/bootstrap/flags.rs

```diff
@@ -545,9 +552,7 @@ Arguments:
                     profile_string.parse().unwrap_or_else(|err| {
                         eprintln!("error: {}", err);
                         eprintln!("help: the available profiles are:");
-                        for choice in Profile::all() {
-                            eprintln!("- {}", choice);
-                        }
+                        eprint!("{}", Profile::all_for_help(" "));
```

Comment:
Personally I liked it better with ` - ` in front, but I don't feel strongly about it.